examples/local/k3s-up.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright 2019 The Vitess Authors.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is an example script that creates a Kubernetes api for topo use by running k3s
+
+set -ex
+cell=${CELL:-'test'}
+
+script_root=$(dirname "${BASH_SOURCE[0]}")
+
+# shellcheck source=./env.sh
+# shellcheck disable=SC1091
+source "${script_root}/env.sh"
+
+k3s server --disable-agent --data-dir "${VTDATAROOT}/k3s/" --https-listen-port "${K8S_PORT}" --write-kubeconfig "${K8S_KUBECONFIG}" > "${VTDATAROOT}"/tmp/k3s.out 2>&1 &
+PID=$!
+echo $PID > "${VTDATAROOT}/tmp/k3s.pid"
+disown -a
+echo "Waiting for k3s server to start"
+sleep 15
+
+# Use k3s built-in kubectl with custom config
+KUBECTL="k3s kubectl --kubeconfig=${K8S_KUBECONFIG}"
+
+#echo "add /vitess/global"
+#$KUBECTL get cm vitess-global || $KUBECTL create configmap vitess-global --from-literal=parent="" --from-literal=contents=""
+
+#echo "add /vitess/$cell"
+#$KUBECTL get cm vitess-$cell || $KUBECTL create configmap vitess-$cell --from-literal=parent="" --from-literal=contents=""
+
+# And also add the CellInfo description for the cell.
+set +e
+echo "add $cell CellInfo"
+"${VTROOT}"/bin/vtctl $TOPOLOGY_FLAGS AddCellInfo \
```

Comment:
The examples no longer have a dependency on `VTROOT`. Please expect `vtctl` to be in the `PATH`. env.sh will dependency check for all the vitess binaries before starting the example, but strangely `vtctl` is missing here:
```
for binary in mysqld etcd etcdctl curl vtctlclient vttablet vtgate vtctld mysqlctl; do
  command -v "$binary" > /dev/null || fail "${binary} is not installed in PATH. See https://vitess.io/docs/get-started/local/ for install instructions."
done;
```

.. do you mind adding it in this PR?

Reply:
Updated